<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电商平台销售数据处理工具v0.4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="fileListTableWidget">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DropOnly</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>状态</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>文件名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>文件路径(可拖入文件/文件夹)</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="addFilesButton">
        <property name="text">
         <string>添加文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeSelectedButton">
        <property name="text">
         <string>删除选中</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectAllButton">
        <property name="text">
         <string>全选/全不选</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="invertSelectionButton">
        <property name="text">
         <string>反选</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;[输出文件夹]内已存在与[将要输出的文件]同名的文件时的处理逻辑&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>输出路径重复文件处理：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="conflictComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;[输出文件夹]内已存在与[将要输出的文件]同名的文件时的处理逻辑&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtBottom</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>重命名</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>覆盖</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>跳过</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="useDirectoryModeCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>指定输入文件夹(切换时清空列表)：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseInputDirButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>选择文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputDirPathLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshDirButton">
        <property name="toolTip">
         <string>重新扫描指定的输入文件夹，并更新下方的文件列表</string>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="outputToSourceCheckBox">
        <property name="text">
         <string>输出到源文件路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>或指定输出路径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseOutputButton">
        <property name="text">
         <string>选择文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputPathLineEdit"/>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startProcessButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>239</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>开始处理</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string extracomment="重新扫描指定的输入文件夹，并更新下方的文件列表"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
